src/pages/console/uibuilder/figmatocode.mdx

```diff
@@ -3,11 +3,11 @@ export const meta = {
   description: "Figma to React code with Amplify Studio",
 };
 
-Amplify Studio offers an integration with Figma, allowing you to generate clean React code by importing your Figma design file. [Figma](https://figma.com/) is a browser-based UI and UX design application that is used to build high-fidelity designs. In the standard product development lifecycle, UI or UX designers build mockups that get implemented as code by developers. Amplify Studio automatically converts any [Figma component](https://help.figma.com/hc/en-us/articles/360038662654-Guide-to-Components-in-Figma) in your Figma file to a [React component](https://reactjs.org/docs/components-and-props.html) that is then usable in your app.
+[Amplify Studio](/console/adminui/start/#logging-in-and-creating-an-app) offers an integration with Figma, allowing you to generate clean React code by importing your Figma design file. [Figma](https://figma.com/) is a browser-based UI and UX design application that is used to build high-fidelity designs. In the standard product development lifecycle, UI or UX designers build mockups that get implemented as code by developers. Amplify Studio automatically converts any [Figma component](https://help.figma.com/hc/en-us/articles/360038662654-Guide-to-Components-in-Figma) in your Figma file to a [React component](https://reactjs.org/docs/components-and-props.html) that is then usable in your app.
```

Comment:
Rather than linking to the login at the first mention of Studio, I recommend moving the link to the first instances of working inside Studio, like inside step 1 below

Reply:
adding it in step 2 makes more sense here